src/ui/Controls/NikseComboBox.cs

```diff
@@ -575,10 +520,10 @@ protected override void OnMouseDown(MouseEventArgs e)
                         totalY += ctl.Top;
                     }
 
-                    if (_listView.Items.Count > 0)
+                    if (listViewItems.Count > 0)
```

Comment:
Don't use `_listView.Count` see https://github.com/SubtitleEdit/subtitleedit/pull/7174/files#r1278103956

listViewItems is the source that is being used to init _lsitView and it's case to check the count from it
directly!